.github/actions/gke/test-config-classic.yaml

```diff
@@ -0,0 +1,15 @@
+# List of test cases for conformance-gke with helm installation options
```

Comment:
Copy paste? )

Reply:
@viktor-kurchenko That's just how the file is in `main`, it was probably a copy-paste issue there. If you want to fix it, let's fix in `main` :p